quickstart/csharp/uwp/keyword-recognizer/README.md

```diff
@@ -0,0 +1,35 @@
+# Quickstart: Recognize keywords in C# under Universal Windows Platform
+
+This sample demonstrates how to recognize keywords with C# under the Universal Windows Platform using the Speech SDK.
+See the [accompanying article](https://docs.microsoft.com/azure/cognitive-services/speech-service/quickstart-csharp-uwp) on the SDK documentation page which describes how to build this sample from scratch in Visual Studio 2017.
+
+## Prerequisites
+
+* A Windows PC with Windows 10 Fall Creators Update (10.0; Build 16299) or later and with a working microphone.
+* [Microsoft Visual Studio 2017](https://www.visualstudio.com/), Community Edition or higher.
+* The **Universal Windows Platform development** workload in Visual Studio.
+* Note: processor target ARM or ARM64 is not yet supported.
```

Comment:
>processor target ARM or ARM64 is not yet supported [](start = 8, length = 50)

is this correct?

Reply:
No, that isn't , missed that part. Will remove.